core/src/test/java/cucumber/runtime/formatter/TimelineFormatterTest.java

```diff
@@ -182,14 +179,14 @@ public void shouldWriteItemsAndGroupsCorrectlyToReportJs() throws Throwable {
     }
 
     private void runFormatterWithPlugin() {
-        try {
-            TestHelper.runFeaturesWithFormatter(Arrays.asList(failingFeature, successfulFeature, pendingFeature), stepsToResult, stepsToLocation,
-                Collections.<AbstractMap.SimpleEntry<String, Result>>emptyList(), Collections.<String>emptyList(),
-                Collections.<Answer<Object>>emptyList(), STEP_DURATION, null, "--plugin", "timeline:" + reportDir.getAbsolutePath());
-        }
-        catch (final Throwable e) {
-            throw new RuntimeException(e);
-        }
+        TestHelper.builder()
```

Comment:
Neat!

Reply:
Thanks. Still not brilliant. The class still has a lot going on as there are so many permutations to handle, it just seemed the best fit. 

Could now break down the class more into constituent parts, if desired, but will do for now

Reply:
Unfortunately my commit had a lot of formatting changes 😱

I really need to change my IntelliJ settings to only format changed lines 🤗

Reply:
Plus, it still doesn't solve the issues of all the "setup cruft" in each test method/class.  Was thinking it would be good to have a test "library" class where pre-prepared scenarios are held and then each test can make use of them.  If they need something slightly different then that can be looked at - but this would take some time to analyse - I will have a little play but it all depends on how much time there is before the PR is, hopefully, merged

at least now the TestHelper is easier to use/deal with